src/CMakeLists.txt

```diff
@@ -1036,8 +1037,7 @@ if (HDF5_GENERATE_HEADERS)
 endif ()
 
 #-----------------------------------------------------------------------------
-# Setup the H5detect utility which generates H5Tinit with platform
-# specific type checks inside
+# Set up source generators
 #-----------------------------------------------------------------------------
 if (HDF5_USE_PREGEN)
```

Comment:
I think this option and related can also be removed.

Reply:
HDF5_GENERATED_SOURCE_DIR can be replaced with HDF5_SRC_BINARY_DIR.

Reply:
I left anything in that looks like it could be needed for H5make_libsettings (which is next on my list, so it'll go away soon).